app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -49,12 +49,23 @@ class PatientComponent extends Component {
 
   searchDemographics(event) {
     event.preventDefault();
-    const { gender, minAge, maxAge, livingStatus } = this.state;
-    let { startDate, endDate } = this.state;
+    const { gender, livingStatus } = this.state;
+    let { startDate, endDate, minAge, maxAge } = this.state;
     const searchParameters = { gender };
     // add appropriate age constraints to the search parameters
     if (minAge || maxAge) {
       if (minAge && maxAge) {
+        // coerce the values of minAge and maxAge to numbers for comparison    
+        minAge = Number(minAge);
+        maxAge = Number(maxAge);
+
+        // switch the min and max ages if the min age is greater than the max age
+        if (minAge > maxAge) {
+          const minAgeStore = minAge;
```

Comment:
@andela-jomadoye How do you prevent a user from inputing a minAge that is higher than maxAge?

* Currently, user input is gotten from a text box, and our application has no control over what the user decides to put in there.
* If we decide to change the input type to be a select box, how are you sure that the user will select a min age lower than the max age?

If there is any better approach you know to solving this problem, please drop it here so I can quickly look into it.